homeassistant/components/tuya/config_flow.py

```diff
@@ -82,7 +83,11 @@ def _try_login(user_input):
         if project_type == ProjectType.INDUSTY_SOLUTIONS:
             response = api.login(user_input[CONF_USERNAME], user_input[CONF_PASSWORD])
         else:
-            api.endpoint = TUYA_ENDPOINT_BASE
+            if user_input[CONF_COUNTRY_CODE] in COUNTRY_CODE_CHINA:
```

Comment:
```suggestion
            if user_input[CONF_COUNTRY_CODE] == COUNTRY_CODE_CHINA:
```

Reply:
It’s an array, so it doesn’t make sence to chnage it to ==

Reply:
The other suggestion above changed the constant to a string.